doc/cli/vast-dump.md

```diff
@@ -0,0 +1,9 @@
+The `dump` command prints configuration options as JSON. An optional `--yaml`
+flag switches the output format to YAML.
+
+For example, to see the amalgamated concept definitions, use the following
+command:
```

Comment:
I'm questioning that "amalgamated" is easier to understand than "irreconcilable". 😉

Reply:
```suggestion
For example, to see all registered concept definitions, use the following
command:
```